demo/nodejs/file.js

```diff
@@ -28,6 +28,10 @@ program
     "AccessKey obtain from the Picovoice Console (https://console.picovoice.ai/)"
   )
   .requiredOption("-i, --input_audio_file_path <string>", "input wav file")
+  .option(
+    "-d, --device <string>",
+    "device to run demo"
+  )
```

Comment:
check you one of the already released products for what helper text to use. Also I don't see the `show_inference_device` option. You need to add that functionality to both demos